wurst/event/DamageModify.wurst

```diff
@@ -84,11 +85,15 @@
 
 			d.releaseInstance()
 
+		init
```

Comment:
this file now has two init blocks?

Reply:
No, it's two different examples, and they're both just comments after all